src/Catalyst.Core.Lib.Tests/UnitTests/Extensions/UInt256ExtensionsTests.cs

```diff
@@ -57,6 +61,34 @@ public void UInt256_ToByteString_And_Back_Should_Keep_Value_Intact(UInt256 bigIn
             xAgain.S2.Should().Be(x.S2);
             xAgain.S3.Should().Be(x.S3);
         }
+
+        [Theory]
+        [ClassData(typeof(UInt256ConversionTestData))]
+        public void Hex_To_UInt256(UInt256 bigInt)
+        {
```

Comment:
There is a method on UInt256 -> CreateFromBigEndian

Reply:
Yeah I was trying to debug a issue, but turned out the bytes were wrong coming through the front end, Stephen ended up fixing it, so reverting this file. Was just for testing.